virtwho/virt/virt.py

```diff
@@ -596,6 +596,12 @@ def _send_data(self, data_to_send):
         @param data_to_send: A dict of source_keys, report
         @type: dict
         """
+
+        if not isinstance(data_to_send, ErrorReport):
+            # Ping all hypervisors for this reporter to update check in time regardless of change
```

Comment:
I would call this only in case, when `self.dest.has_capability("hypervisors_heartbeat") is True`. When the candlepin server does not have the capability, then debug message is little bit confusing now.

Reply:
removed debug method